dbt/models/default/default.vw_pin_land.sql

```diff
@@ -14,7 +14,19 @@ WITH total_influ AS (
         FIRST_VALUE(land.sf)
             OVER (PARTITION BY land.parid, land.taxyr ORDER BY land.lline)
             AS sf_top,
-        SUM(CASE WHEN land.influ IS NULL THEN 0 ELSE 1 END)
+        SUM(
+            CASE
+                WHEN
+                    (
+                        -- The field for this column changed in 2024 to allow
+                        -- greater proration precision.
+                        (land.influ IS NULL AND land.taxyr < '2024')
+                        OR (land.allocpct IS NULL AND land.taxyr >= '2024')
```

Comment:
**suggestion:** I would maybe add the issue description here to provide larger context for _why_ this conditional is needed.